nebula/ui/components/VPNConnectionInfoContent.qml

```diff
@@ -103,21 +78,71 @@ VPNFlickable {
                     Layout.rightMargin: VPNTheme.theme.windowMargin
                 }
 
+                VPNCallout {
+                    calloutCopy: VPNl18n.ConnectionInfoMultiHopCallout
+                    calloutImage: "qrc:/nebula/resources/connection-info.svg"
+                    color: VPNTheme.theme.white
+                    fontPixelSize: VPNTheme.theme.fontSizeSmall
+                    iconSize: VPNTheme.theme.iconSize * 1.25
+                    spacing: VPNTheme.theme.listSpacing * 0.5
+                    visible: serverLocations.isMultipHop
+
+                    Layout.bottomMargin: VPNTheme.theme.windowMargin
+                    Layout.leftMargin: VPNTheme.theme.windowMargin
+                }
+
                 ColumnLayout {
+                    property bool isMultipHop: (typeof(VPNCurrentServer.entryCountryCode) !== undefined
+                        && VPNCurrentServer.entryCountryCode !== "")
+                    id: serverLocations
+
                     spacing: 0
 
+                    Layout.fillWidth: true
                     Layout.leftMargin: VPNTheme.theme.windowMargin
                     Layout.rightMargin: VPNTheme.theme.windowMargin
 
-                    VPNConnectionInfoItem {
-                        title: VPNServerCountryModel.getLocalizedCountryName(
-                            VPNCurrentServer.exitCountryCode
-                        )
-                        subtitle: VPNCurrentServer.localizedCityName
-                        iconPath: "qrc:/nebula/resources/flags/"
-                            + VPNCurrentServer.exitCountryCode.toUpperCase()
-                            + ".png"
-                        isFlagIcon: true
+                    RowLayout {
```

Comment:
Can we consolidate the `isMultiHop` visibility conditionals by just adding `visible: ....isMultiHop` to the `RowLayout {}` ?

Reply:
Unfortunately, we can’t hide the entire row here since we’ll still have to show the last item for single hop.